torch/_inductor/comms.py

```diff
@@ -342,6 +342,148 @@ def reorder_compute_and_comm_for_overlap(
     return order
 
 
+def remove_fsdp2_unsharded_param_graph_input_usage(graph: torch.fx.Graph):
+    """
+    This FX graph pass replaces uses of FSDP2 unsharded params with their corresponding
+    graph intermediates that were fsdp.copy_ into the unsharded params in the original graph.
```

Comment:
Just a nit: but maybe add a code link in the comments to here, from the place in the FSDP code where we insert the `fsdp.copy_`? since someone reading the FSDP code will probably think that the copy_ is inefficient (so we can link them straight to where we try to optimize it away)

Reply:
sounds great! done